code/rnnslu.py

```diff
@@ -381,10 +389,10 @@ def main(param=None):
         if param['clr'] < 1e-5:
             break
 
-    print('BEST RESULT: epoch', param['be'],
-          'valid F1', param['vf1'],
-          'best test F1', param['tf1'],
-          'with the model', folder)
+    print(('BEST RESULT: epoch', param['be'],
+           'valid F1', param['vf1'],
+           'best test F1', param['tf1'],
+           'with the model', folder))
```

Comment:
I think you need to remove the additional set of `()`, otherwise the whole thing gets printed as a tuple, rather than all the values being concatenated.